signing_service/signing_service/signing_service.py

```diff
@@ -268,6 +273,25 @@ def _handle_connection(
             # If received frame is correct, validate transaction.
             else:
                 golem_message_response = self._get_signed_transaction(middleman_message.payload)
+                if isinstance(golem_message_response, SignedTransaction):
+                    transaction_sum_so_far = self._get_signing_service_daily_transaction_sum_so_far()
+                    transaction_sum_combined = transaction_sum_so_far + middleman_message.payload.value
+                    email_notifier = Notifier()
```

Comment:
How about creating instance of `Notifier` in `__init__` ? This way it will be done only once.

Reply:
Done